src/utils/queue.rs

```diff
@@ -3,6 +3,26 @@
 use crate::error::Result;
 use postgres::Connection;
 
+/// The default queue priority
+const DEFAULT_PRIORITY: i32 = 0;
```

Comment:
I think this is pretty clear from the variable and file name.

```suggestion
const DEFAULT_PRIORITY: i32 = 0;
```